gradle/test-intellij.versions.toml

```diff
@@ -1,2 +1,2 @@
 [versions]
-versionList = "IIC-232.8660.185,IIC-223.8836.41"
+versionList = "IIC-233.13135.103,IIC-223.8836.41"
```

Comment:
We should periodically ensure we're testing with the latest released version. Grabbed the version from https://www.jetbrains.com/idea/download/other.html.